packages/stack/compiler/src/index.ts

```diff
@@ -1,7 +1,11 @@
-import {Callback, CompilerPluginObject, Embark, Plugins} /* supplied by @types/embark in packages/embark-typings */ from "embark";
+import { Callback, CompilerPluginObject, Embark, Plugins /* supplied by @types/embark in packages/embark-typings */ } from "embark";
 import { __ } from "embark-i18n";
+import * as fs from "fs-extra";
```

Comment:
We should use Embark's `fs`.

Reply:
Embark's `fs` doesn't do that anymore... long story re: things not working with production installs when we were trying to get 4.0 out the door way back when.

Reply:
Nevermind, I thought you were referring to something else. :D Sure, I can use the fs that hangs off of the embark object.

Reply:
Done.